packages/commerce-sdk-react/src/hooks/ShopperOrders/mutation.test.tsx

```diff
@@ -0,0 +1,155 @@
+import nock from 'nock'
+
+const {withMocks} = mockHttpResponses({directory: path.join(__dirname, '../../../mock-responses')})
+
+const BASKET_ID = 'a10ff320829cb0eef93ca5310a'
+
+const OrderMutationComponent = () => {
+    //Log into registered account when the component is mounted
+    const loginRegisteredUser = useShopperLoginHelper(ShopperLoginHelpers.LoginRegisteredUserB2C)
+    const createOrder = useShopperOrdersMutation(ShopperOrdersMutations.CreateOrder)
+    const createBasket = useShopperBasketsMutation(ShopperBasketsMutations.CreateBasket)
+
+    React.useEffect(() => {
+        loginRegisteredUser.mutate({username: 'alex@test.com', password: 'Test1234#'})
+    }, [])
+
+    //Need to create a prepared basket in order to create an order with basketID
+    const createOrderFlow = async () => {
+        const {data} = await createBasket.mutateAsync({body: {}})
+        createOrder.mutate({
+            body: {basketId: BASKET_ID}
+        })
+}
+
+    return (
+        <>
+            {loginRegisteredUser.isLoading ? (
+                <span>Logging in...</span>
+            ) : (
+                <div>Logged in as {loginRegisteredUser?.variables?.username}</div>
+            )}
+            <div>
+                <button
+                    onClick={() =>
+                        createOrderFlow()
+                    }
+                >
+                    Create Order
+                </button>
+
+                {createOrder.error?.message && (
+                    <p style={{color: 'red'}}>Error: {createOrder.error?.message}</p>
+                )}
+                <hr />
+                <div>{JSON.stringify(createOrder)}</div>
+            </div>
+        </>
+    )
+}
+
+const tests = [
+    {
+        hook: 'createOrder',
+        cases: [
+            {
+                name: 'success',
+                assertions: withMocks(async () => {
+                    renderWithProviders(<OrderMutationComponent />)
+
+                    await waitFor(() => screen.getByText(/alex@test.com/))
+                    await waitFor(() =>
+                        screen.getByRole('button', {
+                            name: /create order/i
+                        })
+                    )
+
+                    // Mocking the server request
+                    nock('http://localhost:3000')
+                        .post((uri) => {
+                            return uri.includes('/checkout/shopper-baskets/')
+                        })
+                        .reply(200, {
+                            basketId: BASKET_ID,
+                            customerInfo: {
+                                email: 'alex@test.com'
+                            }
+                        })
+
+                    // Mocking the server request
+                    nock('http://localhost:3000')
+                        .post((uri) => {
+                            return uri.includes('/checkout/shopper-orders/')
+                        })
```

Comment:
This could be moved to a more localized / globalized location where any test that wants to mock out  can just pass in a payload like this that is expected

```
{
                            orderNo: '000000410',
                            customerInfo: {
                                email: 'alex@test.com'
                            }
```
In another file (that's in a shared location similar to a `utils` dir) you could do. Whether this is useful depends on how much we expect these methods to be called by sibling / adjacent test suties

```
const nockShopperOrders = (mockResponse) => {
 nock('http://localhost:3000')
                        .post((uri) => {
                            return uri.includes('/checkout/shopper-orders/')
                        })
                        .reply(200, mockResponse)
}
```